tembo-cli/src/cmd/apply.rs

```diff
@@ -838,6 +856,41 @@ fn get_postgres_config(
     postgres_config
 }
 
+async fn get_loadable_libraries(
+    mut shared_preload_libraries: Vec<Library>,
+    maybe_extensions: Option<&Vec<ControllerExtension>>,
+) -> Result<Vec<Library>, anyhow::Error> {
+    if let Some(extensions) = maybe_extensions {
+        let trunk_projects_url =
+            "https://registry.pgtrunk.io/api/v1/trunk-projects?extension-name=";
+
+        for ext in extensions.iter() {
```

Comment:
I’m not sure but I think this logic is not handling the case of other trunk projects with matching extension names. Some different trunk projects have extensions of the same name

Reply:
I’m saying when you look up by extension name, you might also get other unrelated trunk projects with extensions of the same name

Reply:
Please check the latest commit to see if it makes sense to you.